### PR TITLE
update eslint-config-google to 0.11.0

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

**Comment by sebastianbenz** [comment]
@ithinkihaveacat PTAL as Andre is OOO

**Comment by sebastianbenz** [comment]
1. Yes, this seems to be a lerna best practices. Added it to gitignore.
2. Made things worth (and better). All tests are moved to the root folder. Removes lots of duplication and speeds things up. It becomes harder though to run package specific tests.
3. Yes.

File: .travis.yml [hunk 1]
```diff
@@ -6,5 +6,9 @@ node_js:
 sudo: required
 addons:
   chrome: stable 
-install: npm install -g npx && npx lerna bootstrap  
+cache:
+  directories:
+    - ~/.npm
+before_install: if [[ `npm -v` != 6* ]]; then npm i -g npm@6; fi
```
Comment (ithinkihaveacat) [inline .travis.yml]: Maybe should document that npm 6 is required for local install as well. Might be possible to use the "prepare" script or something to actually test for this instead.  FFS `npm`…

File: package.json [hunk 2]
```diff
@@ -1,7 +1,48 @@
 {
   "name": "root",
   "private": true,
+  "scripts": {
+    "build": "lerna run build",
```
Comment (ithinkihaveacat) [inline package.json]: Should this be `npx lerna run build`? It's not otherwise installed.

Comment (sebastianbenz) [reply package.json]: done

File: .travis.yml (hunk 1 above)
Comment (sebastianbenz) [reply .travis.yml]: I plan to document the dev setup in another PR.

**Comment by sebastianbenz** [comment]
Thanks Michael! Should all be fixed now. PTAL

**Pull request merged**